content/scout/policy/index.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Get started with policy in Docker Scout
+keywords: scout, supply chain, vulnerabilities, packages, cves, policy
+description: |
+  Policies in Docker Scout lets you define supply chain rules and thresholds
+  for your artifacts, and track how your artifacts perform against those
+  requirements over time
+---
+
+> **Early Access**
+>
+> Docker Scout is in [early access](/release-lifecycle/#early-access-ea). The
+> policy feature is a closed access feature of Docker Scout, and is only
+> available to organizations participating in the early access program for
+> policies.
+>
+> If you're interested in trying out this feature, reach out using the form on
+> the [Docker Scout product page](https://docker.com/products/docker-scout)
+{ .restricted }
+
+In software supply chain management, maintaining the security and reliability
+of artifacts is a top priority. Policy in Docker Scout introduces a layer of
+control, on top of existing analysis capabilities. It lets you define supply
+chain rules for your artifacts, and helps you track how your artifacts perform,
+relative to your rules and thresholds, over time.
+
+Learn how you can use policy to ensure that your artifacts align with your
+desired standards.
+
+## Policy evaluation
+
+When you activate Docker Scout for a repository, image tags that you push are
+[automatically analyzed](../advanced-image-analysis.md). The analysis gives you
+insights about the composition of your images, including what packages that
+they contain and what vulnerabilities that they're exposed to. Policy builds on
+top of the image analysis feature, interpreting the analysis results against
+the rules defined by policies.
+
+A policy defines one or more criteria that your artifacts should fulfill. For
+example, one of the default policies in Docker Scout is the **Critical
+vulnerabilities** policy, which proclaims that your artifacts must not contain
+any critical vulnerabilities. If an artifact contains one or more
+vulnerabilities with a critical severity, that artifact will have failed the
+policy check.
+
+In Docker Scout, policies are not meant to be used for gate-keeping. The
```

Comment:
This is interesting. I think policies _can_ be used for some level of gate-keeping, but not solely gate-keeping. A lot of other tools often only provide pass/fail information, which limits what you can do. Our implementation allows ratcheting as a method of adoption, but you could still gate-keep based on those results, and indeed, once you have managed to get a policy being met across an org, you probably would want to gate-keep in order to keep things that way.